lib/Backends/BackendUtils.cpp

```diff
@@ -145,6 +145,7 @@ runtime::RuntimeBundle runtime::RuntimeBundle::create(const Function &F) {
     symbol.output = isOutput(V);
     symbol.input = isInput(V);
     symbol.symbolCategory = SymbolCategory::Placeholder;
+    symbol.placeholder = V;
```

Comment:
I like this change but we must ensure the Module outlives the RuntimeBundle for this pointer to be valid, is that guaranteed currently?

Reply:
Yes good point, their lifetimes are tied together in HostManager

Reply:
We have usages of the CompiledFunction (+RuntimeBundle) outside of the runtime, can we add a comment somewhere about this restriction. Maybe even on `Backend::compile`?